styles.css

```diff
@@ -24,10 +24,36 @@
 #menu::before {
     left: 0;
     top: -20px;
+    transform: rotate(0deg);
+    transition: ease-in-out .5s;
```

Comment:
De properties transform en transition staan hier niet bij in de uitwerking. Weet het niet zeker, maar volgens mij is rotate hier redundant.